src/Controller/FavoritesController.php

```diff
@@ -143,23 +173,21 @@ public function delete(Request $request) {
     if ($data instanceof JsonResponse) {
       return $data;
     }
+    $data['uid'] = $this->currentUser()->id();
     $favorite_items = $this->entityTypeManager()
       ->getStorage('vy_favorite_item')
-      ->loadByProperties([
-        'uid' => $this->currentUser()->id(),
```

Comment:
Does it mean that I could update other user data from frontend if I know his uid?

Reply:
on 176 line we have `$data['uid'] = $this->currentUser()->id();`